project/dependencies.scala

```diff
@@ -7,7 +7,7 @@ object Dependencies {
 
   val scopt = "com.github.scopt" %% "scopt" % "3.2.0"
   val log4j = "log4j" % "log4j" % "1.2.17"
-  val defaultDispatchVersion = "0.11.2"
+  val defaultDispatchVersion = "0.11.3"
```

Comment:
I think this pull request overall is useful if source change was required due to dispatch/reboot#116, but could we keep the defaultDispatchVersion to 0.11.2?